lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -117,7 +117,7 @@ def setUp(self):
         super(TestInstructorGradeReport, self).setUp()
         self.course = CourseFactory.create()
 
-    @ddt.data([u'student@example.com', u'ni\xf1o@example.com'])
+    @ddt.data([u'student@contrived_example.com', u'ni\xf1o@contrived_example.com'])
```

Comment:
What happens if the email address isn't changed?

Reply:
Failing test:

ERROR: test_unicode_emails_1__u_student_example_com___u_ni_xf1o_example_com__ (lms.djangoapps.instructor_task.tests.test_tasks_helper.TestInstructorGradeReport)

Reply:
Hrmm, does the failure still happen now that we made the exclude optional and _not_ the default?
Have you tried reverting it since we've made that change?

I would expect that any code path that doesn't explicitly set `exclude_fake_email=True` should be unaffected in its tests...